compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensions.cpp

```diff
@@ -929,32 +929,6 @@ void transform_dialect::WorkgroupSwizzleOp::getEffects(
 // TestVectorLayoutAnalysisOp
 //===----------------------------------------------------------------------===//
 
-static void setAnchorOpsFromAttributes(VectorLayoutAnalysis &analysis,
```

Comment:
I prefer this staying in TransformExtensions tests. Is there any reason we shifted this?

Reply:
Yes, I needed a way to set layouts from annotations in the IR and this function was "trapped" inside the  cpp code. What I was imagining was that the user (either through a separate pass or some other way) would set layouts on specific ops and then we would read the layouts and propagate to the rest of the operations. This function here is the first step towards that goal. I didn't want to expand the scope of the PR so I settled for just moving it to a place where it can be used by others.

Reply:
SGTM